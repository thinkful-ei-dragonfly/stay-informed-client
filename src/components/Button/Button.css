.Button {
  padding: 10px 10px;
  font-weight: 700;
  margin:10px 5px 10px 0;
  border: .5px solid lightgray;
  background-color: #fffeff;
  border-radius: 10px;
  box-shadow: 0 5px 5px rgba(0,0,0, .6); 
  text-decoration: none;
  color:#02111B;

}

.Button:hover {
  cursor: pointer;
  box-shadow: 0 5px 5px rgba(179, 177, 177, 0.6); 
}