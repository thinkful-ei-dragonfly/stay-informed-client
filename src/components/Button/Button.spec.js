import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Button from './Button'

describe(`Button component`, () => {
  it('renders a form.Button by default', () => {
    const wrapper = shallow(<Button />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
