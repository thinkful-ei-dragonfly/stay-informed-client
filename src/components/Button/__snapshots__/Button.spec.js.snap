// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component renders a form.Button by default 1`] = `
<button
  className="Button"
/>
`;
